react/more_react_concepts/reducing_state.md

```diff
@@ -85,17 +85,17 @@ about how to use reducers in React and how to refactor `useState` into `useReduc
 
 </div>
 
-### Knowledge Check
+### Knowledge check
 
 This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
 
-*   <a class="knowledge-check-link" href="#what-are-reducers">What are reducers?</a>
-*   <a class="knowledge-check-link" href="https://react.dev/reference/react/useReducer#writing-the-reducer-function">How would you declare a reducer?</a>
-*   <a class="knowledge-check-link" href="#the-usereducer-hook">What does the dispatch function do?</a>
-*   <a class="knowledge-check-link" href="https://react.dev/learn/extracting-state-logic-into-a-reducer#consolidate-state-logic-with-a-reducer">What steps can you follow to migrate from useState to useReducer?</a>
+* <a class="knowledge-check-link" href="#what-are-reducers">What are reducers?</a>
+* <a class="knowledge-check-link" href="https://react.dev/reference/react/useReducer#writing-the-reducer-function">How would you declare a reducer?</a>
+* <a class="knowledge-check-link" href="#the-usereducer-hook">What does the dispatch function do?</a>
+* <a class="knowledge-check-link" href="https://react.dev/learn/extracting-state-logic-into-a-reducer#consolidate-state-logic-with-a-reducer">What steps can you follow to migrate from useState to useReducer?</a>
```

Comment:
```suggestion
- <a class="knowledge-check-link" href="#what-are-reducers">What are reducers?</a>
- <a class="knowledge-check-link" href="https://react.dev/reference/react/useReducer#writing-the-reducer-function">How would you declare a reducer?</a>
- <a class="knowledge-check-link" href="#the-usereducer-hook">What does the dispatch function do?</a>
- <a class="knowledge-check-link" href="https://react.dev/learn/extracting-state-logic-into-a-reducer#consolidate-state-logic-with-a-reducer">What steps can you follow to migrate from useState to useReducer?</a>
```

Reply:
I noticed there are some more of these `*` instead of `-` for list items earlier in the lesson. If you feel like changing those you can do so in a separate PR if you want. The proposed changes I made here are unrelated to this issue as well but I just happened to catch them while reviewing

Reply:
@ManonLef  I'm not sure how those all got changed.  I apologize. I for sure didn't do that manually.